oic.r.roles.raml

```diff
@@ -2,7 +2,7 @@
 title: OICSecurityRolesResource
 version: v1.0-20170323
 schemas:
-  - Cred: !include schemas/oic.r.cred.json
+  - Roles: !include schemas/oic.r.roles.json
```

Comment:
The type of the resource should be oic.r.roles. It contains a single property, "roles", which is an element of oic.sec.cred.